components/AppNav.tsx

```diff
@@ -124,15 +97,25 @@ const AppNav: React.FC<{}> = () => {
     // TODO: replace with typing
     const username = _.get(session, 'user.username', '')
 
-    return <LoggedInAuthNav username={username} />
+    return <LoggedInAuthNav username={username} session={session} />
```

Comment:
This component grabs the isAdmin value that `AppNav` component already grabs, and there isn't any other logic other than returning the ProfileDropdownMenu component. What is the benefit of having this `LoggedInAuthNav` component?

Reply:
You're right about that!